src/game/client/water/w3dwatertracks.cpp

```diff
@@ -13,15 +13,322 @@
  *            LICENSE
  */
 #include "w3dwatertracks.h"
+#include "assetmgr.h"
+#include "baseheightmap.h"
+#include "camera.h"
 #include "dx8indexbuffer.h"
 #include "dx8vertexbuffer.h"
+#include "dx8wrapper.h"
+#include "filesystem.h"
+#include "globaldata.h"
+#include "rinfo.h"
+#include "rtsutils.h"
+#include "shadermanager.h"
+#include "terrainlogic.h"
+#include "vertmaterial.h"
+#include "w3dshroud.h"
+#include "water.h"
 #ifdef GAME_DLL
 #include "hooker.h"
 #endif
+

```

Comment:
For VS17 which i still use to compile needs #include <cstdio> or <stdio.h> for file functions like fopen

Reply:
fixed.